search-app/src/cmr/search/services/parameters/conversion.clj

```diff
@@ -27,6 +27,7 @@
    :version :string
    :updated-since :updated-since
    :revision-date :revision-date
+   :created-at :revision-date
```

Comment:
This will not work on the options parameter for `created-at`. Also, we should probably rename `:revision-date` to a better name.